src/Gui/Application.cpp

```diff
@@ -1947,13 +1947,11 @@ void Application::runApplication(void)
     // http://forum.freecadweb.org/viewtopic.php?f=3&t=15540
     mainApp.setAttribute(Qt::AA_DontShowIconsInMenus, false);
 
-#ifdef Q_OS_UNIX
```

Comment:
Have you tested it that dropping this check doesn't cause problems on other OSes?

Reply:
Nope. I checked in possible extent that the function exists on other OSes.
I wish the CI is building on all OSes but as it's not the case anymore, have to bet on beta users. :)
Do you think we should ask to alpha users ?